cppForSwig/BlockUtils.h

```diff
@@ -42,7 +42,7 @@
 #ifdef _MSC_VER
    #include "mman.h"
    #include "leveldb_windows_port\win32_posix\win32_posix.h"
-   #else
+   #elif !defined(__MINGW32__)
```

Comment:
Formatting nitpick: I'd move this back to the beginning of the line just so that the ifdef/elif delineations are clearer.